src/button-group/menu-dropdown-item.tsx

```diff
@@ -0,0 +1,77 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import React from 'react';
+import clsx from 'clsx';
+
+import InternalButton from '../button/internal';
+import { ButtonDropdownProps } from '../button-dropdown/interfaces';
+import ButtonDropdown from '../button-dropdown/internal';
+import Tooltip from '../internal/components/tooltip';
+import { CancelableEventHandler, fireCancelableEvent } from '../internal/events';
+import { ButtonGroupProps } from './interfaces';
+
+import styles from './styles.css.js';
+import testUtilStyles from './test-classes/styles.css.js';
+
+const MenuDropdownItem = React.forwardRef(
+  (
+    {
+      item,
+      showTooltip,
+      onItemClick,
+      expandToViewport,
+    }: {
+      item: ButtonGroupProps.MenuDropdown;
+      showTooltip: boolean;
+      onItemClick?: CancelableEventHandler<ButtonGroupProps.ItemClickDetails>;
+      expandToViewport?: boolean;
+    },
```

Comment:
Suggestion (readability): You could extract the prop definitions in a separtate `MenuDropdownItemProps` interface.

Reply:
Extracteed MenuDropdownItemProps to interfaces.ts